cfgov/diversity_inclusion/forms.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import division
+
+from django import forms
+
+
+# Form input attributes for Capital Framework compatibility.
+# Technique copied from data_research/forms.py
```

Comment:
Could this be centralized somewhere, to avoid duplication?

Reply:
We could define `text_input_attrs` anywhere and have it imported into these two forms, but what would be the appropriate place for that definition?

Reply:
Maybe something like core/widgets.py?